package.json

```diff
@@ -24,11 +24,14 @@
     "start:web": "office-addin-debugging start manifest.xml web",
     "stop": "office-addin-debugging stop manifest.xml",
     "validate": "office-addin-manifest validate manifest.xml",
-    "watch": "webpack --mode development --watch"
+    "watch": "webpack --mode development --watch",
+    "generate": "graphql-codegen --config codegen.yml"
```

Comment:
You probably want to run generate as part of `postinstall` so that users always have the most recent version of the graphql schema / typing info